module/metrics.go

```diff
@@ -424,3 +424,29 @@ type PingMetrics interface {
 	// NodeInfo tracks the software version, sealed height and hotstuff view of a node
 	NodeInfo(node *flow.Identity, nodeInfo string, version string, sealedHeight uint64, hotstuffCurView uint64)
 }
+
+type HeroCacheMetrics interface {
+	// BucketAvailableSlots keeps track of number of available slots in buckets of cache.
+	BucketAvailableSlots(uint64, uint64)
+
+	// OnSuccessfulWrite is called whenever a new entity is successfully added to the cache.
+	OnSuccessfulWrite()
+
+	// OnEntityEjectedAtFullCapacity is called whenever adding a new entity to the cache results in ejection of another entity.
+	// This normally happens when the cache is full.
+	OnEntityEjectedAtFullCapacity()
+
+	// OnEmergencyKeyEjection is called whenever a bucket is found full and all of its keys are valid.
+	// Hence, adding a new entity to that bucket will replace the oldest valid key inside that bucket.
+	OnEmergencyKeyEjection()
```

Comment:
This `OnEmergencyKeyEjection()` function refers to `Key`, but above the similar `OnEntityEjectedAtFullCapacity()` function refers to `Entity`. Better to have a consistent referring to the same thing?

Reply:
https://github.com/onflow/flow-go/commit/7e88c0596bcc0c137114a5a2d024915a5b481201